src/globals.h

```diff
@@ -11,7 +11,7 @@
 #define P2_EXTEND 0x01
 #define P2_MORE 0x02
 
-#define MAX_MESSAGE_LENGTH 1500  // MTU Size
```

Comment:
@garious was MTU here in reference to what we shred to in core?  I couldn't find any reference in the BOLOS docs for an MTU larger the 64b for USB and 32b for BLE

Reply:
Yeah, that's our max message size and we should try to support that, if possible, not just the messages that we know how to parse.

Reply:
Right. Thanks!  I think it's actually a bit smaller https://github.com/solana-labs/solana/blob/62040cef560331a4f02a8d0a034652056458179e/sdk/src/packet.rs#L9-L13

1500b is like 35% of available SRAM.  I couldn't even add another field without popping the stack.  So went for the quick win :upside_down_face: 

I'll add an issue to track this deficiency for now. I know there are at least a few places we can optimize for space

Reply:
Yeah, use that number then!  Easiest place to get bytes back is the base58 function.